themes/mongodb/src/widgets/deluge/deluge.js

```diff
@@ -21,13 +21,19 @@ function addQueryParameters(url, parameters) {
 class Deluge extends preact.Component {
     constructor(props) {
         super(props);
+
+        const buf = new Uint8Array(16);
+        crypto.getRandomValues(buf);
+
         this.state = {
             'answers': {},
             'emailError': false,
             'formLengthError': false,
-            'voteAcknowledgement': null
+            'voteAcknowledgement': null,
+            'voteId': new TextDecoder('utf-8').decode(buf)
```

Comment:
TextDecoder isn't available in IE.

This should work to get a Base64 string:
```
btoa(Array.prototype.map.call(buf, function (ch) {
        return String.fromCharCode(ch);
    }).join(''));
```

You'll get a bonus `==` suffix (due to base64 padding) which you should strip off